.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: go
 
 go:
-  - go1.8
```

Comment:
Removing 1.8 which is failing now with

```
$ go get -u github.com/golang/lint/golint
# golang.org/x/tools/go/internal/gcimporter
../../../golang.org/x/tools/go/internal/gcimporter/bexport.go:212: obj.IsAlias undefined (type *types.TypeName has no field or method IsAlias)
```

Reply:
Probably worth dropping 1.9 too, I normally keep the latest 2 released versions + tip

Reply:
@campoy Updated.